.github/workflows/build.yml

```diff
@@ -1,5 +1,8 @@
 name: build
 
+env:
+  COSMOCC_VERSION: 3.9.2
```

Comment:
N.B. this creates another place that has to be updated when a new cosmocc dist is rolled.

Another option would be to have a `.cosmoversion` or something that has the version number in it and read that for the Makefile as well as this action.